INSTALL.md

```diff
@@ -98,7 +98,15 @@ or want to develop, you need to follow those steps.
 
 On Debian-based systems:
 
-   apt install python3-pyqt5 python3-venv python3-dev
+    apt install python3-pyqt5 python3-venv python3-dev
+
+For discid support (optional):
+
+    apt install python3-libdiscid
```

Comment:
I would leave the Python wrapper to the pip install. What the user should install is the libdiscid0 package for the actual library.

Reply:
Ah yes, fixed and rebased